packages/dd-trace/src/plugins/tracing.js

```diff
@@ -101,14 +101,15 @@ class TracingPlugin extends Plugin {
     }
   }
 
-  startSpan (name, { childOf, kind, meta, metrics, service, resource, type } = {}, enter = true) {
+  startSpan (name, { childOf, kind, meta, metrics, service, resource, type, startTime } = {}, enter = true) {
     const store = storage('legacy').getStore()
     if (store && childOf === undefined) {
       childOf = store.span
     }
 
     const span = this.tracer.startSpan(name, {
       childOf,
+      startTime, // allow custom start time
```

Comment:
I don't think we should generally add the startTime, since this is only used in Prisma.

Could the startTime also not just be a tag instead?

Reply:
I don't believe adding startTime as a tag would work here. When we create a new span we either use the provided startTime or get the current time `this._startTime = fields.startTime || this._getTime()`. 

The other option I can think of if we don't want to add a startTime to the tracing plugin is to call the downstream `tracer.startSpan` directly from the prisma plugin since it already accepts a startTime field - https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/opentracing/tracer.js#L50.